### DISCUSSION TITLE
Installing problem

### DISCUSSION THREAD

**Discussion by hanori** [microsoft/DTrace-on-Windows#21] [General]
The link "https://download.microsoft.com/download/B/D/4/BD4B95A5-0B61-4D8F-837C-F889AAD8DAA2/DTrace.amd64.msi" in readme.md fails to install.
However, https://www.microsoft.com/en-us/download/details.aspx?id=100441 in DTrace.amd64.msi will install successfully.

I think you should update the readme.md.

**Comment by deivard** [comment]
Thank you for this discussion! I was experiencing the same problem when using this link: https://download.microsoft.com/download/B/D/4/BD4B95A5-0B61-4D8F-837C-F889AAD8DAA2/DTrace.amd64.msi. I could not finish installing due to the DTrace service not being able to start even though I have admin privileges. With the second link, DTrace was installed successfully.
